src/libraries/Common/src/Interop/Unix/System.Globalization.Native/Interop.Normalization.cs

```diff
@@ -12,7 +13,19 @@ internal static partial class Globalization
         [DllImport(Libraries.GlobalizationNative, CharSet = CharSet.Unicode, EntryPoint = "GlobalizationNative_IsNormalized")]
         internal static extern int IsNormalized(NormalizationForm normalizationForm, string src, int srcLen);
```

Comment:
Nit: Change this to take `char*` too so that the APIs are symmetric?

Reply:
Changed for both platforms